spec/models/provider_spec.rb

```diff
@@ -39,7 +39,7 @@
           provider: "govuk",
           uid: auth_subject_uid,
           info: {
-            first_name: "first", last_name: "last", email: "test@test.com", description: "desc", roles: "a,b"
+            first_name: "first", last_name: "last", email: "provider@test.com", description: "desc", roles: "a,b"
```

Comment:
We should really change all domains to @example.com as it is reserved for testing by IANA

Maybe one for another time